### PR TITLE
fixed typo in hauler store save

### PR DESCRIPTION
**Pull request by zackbradys**
**Please check below, if the PR fulfills these requirements:**
- [X] Commit(s) and code follow the repositories guidelines.
- [X] Test(s) have been added or updated to support these change(s).
- [X] Doc(s) have been added or updated to support these change(s).

**Associated Links:**

- N/A

**Types of Changes:**

- Bugfix

**Proposed Changes:**

- Fixed typo in the `platform` flag for `hauler store save`

**Verification/Testing of Changes:**

- N/A

**Additional Context:**

- N/A

### CODE REVIEW

File: internal/flags/save.go [hunk 1]
```diff
@@ -12,5 +12,5 @@ func (o *SaveOpts) AddFlags(cmd *cobra.Command) {
 	f := cmd.Flags()
 
 	f.StringVarP(&o.FileName, "filename", "f", "haul.tar.zst", "(Optional) Specify the name of outputted archive")
-	f.StringVarP(&o.Platform, "platform", "p", "", "(Optional) Specifiy the platform of the images for the outputted archive... i.e. linux/amd64 (defaults to all)")
+	f.StringVarP(&o.Platform, "platform", "p", "", "(Optional) Specify the platform for runtime imports... i.e. linux/amd64 (defaults to all)")
```
Comment (dweomer) [inline internal/flags/save.go]: this is dancing right on the line of ambiguity. i feel like "defaults to all" ➡️ "the default value, an empty string, implies all platforms" but even that is too wordy 🤔

Comment (dweomer) [inline internal/flags/save.go]: hmm, maybe:

```suggestion
	f.StringVarP(&o.Platform, "platform", "p", "", "(Optional) Specify the platform for runtime imports... i.e. linux/amd64 (unspecified implies all)")
```

Comment (amartin120) [reply internal/flags/save.go]: I like that.

**Review by amartin120 — APPROVED** [review]
LGTM
